Tasks/Common/azure-arm-rest/azure-arm-resource.ts

```diff
@@ -12,7 +12,7 @@ export class ResourceManagementClient extends azureServiceClient.ServiceClient {
     constructor(credentials: msRestAzure.ApplicationTokenCredentials, subscriptionId: string, options?: any) {
         super(credentials, subscriptionId);
 
-        this.apiVersion = '2017-05-10';
+        this.apiVersion = '2016-06-01';
```

Comment:
You shouldn't reduce it. We have updated the version due to a new feature in the new version of the API. I would recommend passing a new isAzureStackEnvironment Boolean in the options object and based on that we can select the API-version